static/js/subs.js

```diff
@@ -1043,10 +1043,9 @@ $(function () {
 
         loading.make_indicator(indicator_elem);
 
-        channel.post({
-            url: "/json/get_subscribers",
+        channel.get({
+            url: "/json/streams/" + stream + "/members",
```

Comment:
I think that this is going to be a problem for stream names that have a `/` in them.  We probably need to do some sort of URL-encoding of the stream name here...  

@lfaraone for his thoughts on this one.